packages/next-ui/Snackbar/DismissibleSnackbar.tsx

```diff
@@ -0,0 +1,31 @@
+import { MessageSnackbar } from './MessageSnackbar'
+import { MessageSnackbarProps } from './MessageSnackbarImpl'
+
+export type DismissibleSnackbarProps = MessageSnackbarProps & {
+  id?: string
+  storageType?: 'localStorage' | 'sessionStorage'
+}
+export function DismissibleSnackbar(props: DismissibleSnackbarProps) {
+  const { storageType = 'localStorage', id, onClose, ...rest } = props
+  const messageId = `MessageSnackBar_${id}`
```

Comment:
```suggestion
  const messageId = `MessageSnackBar_${id}`
  const [open, setOpen] = useState(false)

  // since we're syncing with an external system: useEffect
  useEffect(() => {
     setOpen(globalThis[storageType]?.getItem(messageId))
  }, [messageId])
```